src/main/java/hudson/plugins/ec2/win/EC2WindowsLauncher.java

```diff
@@ -8,34 +10,29 @@
 import hudson.remoting.Channel;
 import hudson.remoting.Channel.Listener;
 import hudson.slaves.ComputerLauncher;
-
 import java.io.IOException;
 import java.io.OutputStream;
 import java.io.PrintStream;
 import java.nio.charset.StandardCharsets;
 import java.util.concurrent.TimeUnit;
-
 import jenkins.model.Jenkins;
 import org.apache.commons.io.IOUtils;
 
-import com.amazonaws.AmazonClientException;
-import com.amazonaws.services.ec2.model.Instance;
-
 public class EC2WindowsLauncher extends EC2ComputerLauncher {
+
     private static final String SLAVE_JAR = "slave.jar";
-    
-    final long sleepBetweenAttemps = TimeUnit.SECONDS.toMillis(10);
+    private static final long DEFAULT_RETRY_DELAY_SECONDS = 10;
 
```

Comment:
I like the constant, but let's also keep track of the ms (as a constant or field, does not matter) here. See comment below.